int64 clock 
int8 Duration 
int8 SuperDuration 

int8 bGameEnd 

int8 CurAction 
int8 CurGame 

int8 SuperObj_Num 
int16 SuperObj_X 
int16 SuperObj_Y 

int8 Teleport 
int8 LoadedObjects 

int16 US_Front 
int16 US_Left 
int16 US_Right 

int16 CSLeft_B 
int16 CSLeft_G 
int16 CSLeft_R 

int16 CSRight_B 
int16 CSRight_G 
int16 CSRight_R 

int16 PositionX 
int16 PositionY 
int16 Compass 

int8 TM_State 

int64 Time  
int8 WheelLeft 
int8 WheelRight 

int8 LED_1 
int8 MyState 
